contributing/diversity/governance.rst

```diff
@@ -0,0 +1,110 @@
+Diversity Initiative Governance
+===============================
+
+Membership
+----------
+
+Membership of Symfony’s Diversity Initiative is open to any member of the
+Symfony community; to avoid the risk of elitism or meritocracy, no requirement
+is needed to be involved. All members, at any time, are invited to put forward
+ideas and suggestions as a proposal for an actionable item.
+
+Guidance
+--------
+
+The project leader, Fabien Potencier, is responsible for publicly appointing
+five (5) key players of the initiative to provide guidance and drive it forward,
+but also retains the right to revoke any of the key players at any time. These
+guiding key players should:
+
+* Be committed to the initiative’s cause and have joined because they want to
+  help the initiative to deliver its purpose most effectively for the
+  community’s benefit.
+* Recognize that meeting the initiative’s purpose is an ongoing effort.
+* Be committed to good governance and want to contribute to the initiative’s
+  continued improvement.
+
+Veto
+~~~~
+
+The project leader (Fabien Potencier) will have the right to veto any actionable
+item, regardless of the vote of the initiative’s key players. The project leader
+may, at their discretion, also appoint other people from among the initiative’s
+key players to also have the right to veto - in such a case these people are
+expected to use appropriate judgement to know when to use a “no” vote or a veto.
+Any single veto will reject an actionable item.
+
+The purpose of having members with the right to veto is to prevent a “people’s
+majority” from overruling the core interests of the Symfony project. This will
+encourage communication between proposing members, the initiative’s key players
+and the Core Team to create realistic proposals, and in return any veto will
+come with a full explanation (not just a justification).
+
+Advice Process
+~~~~~~~~~~~~~~
+
+When a proposal on an actionable item is ready to be decided on, insight from
+the community (advice, general consensus, or non-binding poll) should be
+requested from the wider community - this will aim to include both those who
+will be meaningfully affected and those with meaningful expertise in the matter
+at hand.
+This feedback will enable the key players to have the confidence to vote for the
+best possible decision according to the information they have available, knowing
+that the responsibility they accept for said vote is justified.
+
+Voting
+~~~~~~
+
+Key players have the right to vote on proposals for actionable items.
+For an actionable item to pass, approval from greater than 50% of the voting key
+players is required. Use or management of finances/donations require at least a
+two-thirds majority to pass.
+The quorum required for votes to be considered valid is 75% of active, appointed
```

Comment:
75% of 5 players = 3.75 -> how should we round this? 3 or 4? Also, if we require 4 out of 5 to vote, we could require the 5 of them.

Reply:
My understanding is that greater than 50% of 75% need to vote yes for something to pass.

So, if 4 people vote (2 yes, 2 no) it would be a stalemate and would not pass - the fifth member would be required to be the deciding vote.

Reply:
Requiring 100% is not handy when we ever decide to increase the number of key players or when 1 person is not available.
In this case, 4 players voting could let an item pass with 3 votes for and 1 against.

More than 50% of the votes need to be 'yes'. More than 75% of the key players should vote.

Reply:
@TimoBakx I think we are talking about different things.

The document says: *"to be valid, 75% of total people must vote"*  Total people is 5, so 75% is 3.75. Should we require 3 or 4?

What I added is that if we require 4 out of 5 people to vote, is practically the same as requiring 5. But that's another point. The question is: do we require 3 or 4 people to vote? Thanks!

Reply:
I think the wording should be changed to "to be valid, at least 75% of total people must vote".

Reply:
A minor note: I think the original redaction was nice and precise ... I'd just want to clarify what happens when 75% leads to a non-integer division. Thanks!

Reply:
You're right, @javiereguiluz. Three quarters of a person is... a bad idea. ^_^
Adding the "at least" part would clarify that in either case.

Reply:
Added *at least 75%* to account for non-integer division :+1: